ailab-webapp/webapp/routers/sentiment.py

```diff
@@ -0,0 +1,47 @@
+import traceback
+
+from fastapi import APIRouter
+from ..webapp_models.generic_models import ResultResponse
+
+from models.Sentiment.Analysis import news_analyzer, txt_analyzer, txt_summation
+
+router = APIRouter(
+    prefix="/sentiment",
+    tags=["sentiment"]
+)
+
+
+@router.post("/get_recent_news_sentiment")
+async def get_recent_news_sentiment():
+    try:
+        result = news_analyzer()
+    except Exception as e:
+        return ResultResponse(status=-1, message=f"An exception occurred {str(e)}:\n{traceback.format_exc()}", )
+    return ResultResponse(status=0, result=result)
+
+
+@router.get("/get_single_stock_news_sentiment/{ticker}")
+async def get_single_stock_news_sentiment(ticker):
+    try:
+        result = news_analyzer(ticker)
+    except Exception as e:
+        return ResultResponse(status=-1, message=f"An exception occurred {str(e)}:\n{traceback.format_exc()}", )
+    return ResultResponse(status=0, result=result)
+
+
+@router.get("/get_text_sentiment/{txt_file}")
+async def get_text_sentiment(txt_file):
+    try:
+        result = txt_analyzer(txt_file)
+    except Exception as e:
+        return ResultResponse(status=-1, message=f"An exception occurred {str(e)}:\n{traceback.format_exc()}", )
+    return ResultResponse(status=0, result=result)
+
+
+@router.get("/financial_summary/{txt_file}")
```

Comment:
It may not be a good practice to pass a long paragraph in the url link. Could you please use .post instead?
e.g.
@router.post("/get_text_sentiment")
async def get_text_sentiment(txt_file:str):
    try:
        result = txt_analyzer(txt_file)
    except Exception as e:
        return ResultResponse(status=-1, message=f"An exception occurred {str(e)}:\n{traceback.format_exc()}", )
    return ResultResponse(status=0, result=result)